content/rs/administering/_index.md

```diff
@@ -5,10 +5,24 @@ weight: 50
 alwaysopen: false
 categories: ["RS"]
 ---
-This section covers everything you need to know to get up and running
-with RS, included installing, configuring a cluster, maintaining a
-cluster, creating databases, etc.
+This section covers everything you need to know to get up and running with RS.
 
-Topics include:
+{{% note %}}
+In addition to using the RS web UI to manage the cluster,
+you can also access cluster operations through the cluster REST API.
+
+To access the cluster REST API documentation, either:
+
+- Open the HTML files in the RS software package:
+    1. Download the RS software package.
+    1. Unarchive the package.
+    1. Unarchive the rlec_docs.tar.gz package.
+    1. Open the `rlec_docs\html\index.html` file in a web browser.
+- Open the HTML files in an installed RS node:
+    1. Connect to the terminal of an RS node.
+    1. Change directory to `/opt`.
```

Comment:
Please check with Gideon whether this path is relevant for all supported OS

Reply:
@gidon@redislabs.com Do these instructions look OK to you?

Reply:
@bmansheim once the installer improvements are pushed the /opt path could be configured to another path. I assume there are more mentions of explicit paths in the documentation? if so we need to consider how to handle them in general.

Reply:
@gidon@redislabs.com There are many references to explicit paths in the docs. I'm going to merge this one. Please open a ticket in the context of the installer improvements about updating current docs to remove explicit paths.
cc: @banker